packages/core/server/src/endpoints/ping.ts

```diff
@@ -0,0 +1,17 @@
+import fp from 'fastify-plugin';
+
+import { logger } from '@verdaccio/logger';
+
+function pingPlugin(fastify, _opts, done) {
+  fastify.get('/-/ping', async () => {
+    logger.http('ping endpoint');
+    return {};
+  });
+
+  done();
+}
+
+export default fp(pingPlugin, {
```

Comment:
In this use case, the `fastify-plugin` is not necessary, but it is fine

We should use the `fastify-plugin` only for those "piece of code" that must be shared across other fastify's context
[here more datail](https://stackoverflow.com/a/61054534/3309466)